src/index.js

```diff
@@ -16,6 +16,7 @@ export Flex from './Flex';
 export Formbot from './Form/Formbot';
 export FormError from './Form/FormError';
 export FormGroup from './Form/FormGroup';
+export FieldSet from './Form/FormGroup';
```

Comment:
`Fieldset` should be an entirely new component. FormGroups refer to https://getbootstrap.com/docs/4.0/components/forms/#form-groups which adds bottom margin to adjacent inputs

Reply:
added but `FieldSet` does something similar in the presence of adjacent elements - unclear how one would mix usage of  `FieldSet` and `FormGroup`

Reply:
hm also `web-app` has a different `FieldSet` which has a sublegend and is styled differently - should we leave them as is or theme `FieldSet`s?